### ISSUE TITLE
ui5-segmentedbutton: Some toggle buttons contain ellipsis

### ISSUE THREAD

**Issue by MarcusNotheis** [SAP/ui5-webcomponents#1616]
**Describe the bug**
A set of Toggle Buttons within Segmented Button will in some cases have ellipsis.

**To reproduce**
1. Open the provided codesandbox
2. See the error (in case it is not there reload a couple of times)

**Isolated example**
https://codesandbox.io/s/ui5-webcomponents-ig0mo?file=/index.html
```html
<ui5-segmentedbutton>
    <ui5-togglebutton design="Default">Button 1</ui5-togglebutton>
    <ui5-togglebutton design="Default" pressed>Button 2</ui5-togglebutton>
    <ui5-togglebutton design="Default">Button 3</ui5-togglebutton>
</ui5-segmentedbutton>
```

**Expected behavior**
All text is fully shown

**Screenshots**
![image](https://user-images.githubusercontent.com/8775211/81950884-cc4f3500-9604-11ea-8b52-719764c879a4.png)

**Context**
- UI5 Web Components version: 1.0.0-rc.7
- OS/Platform: macOS Catalina 10.15.4
- Browser (if relevant): Chrome 81.0.4044.138

**Affected components** *(if known)*
- ui5-segmentedbutton

Original Issue: https://github.com/SAP/ui5-webcomponents-react/issues/513

**Comment by MarcusNotheis** [comment]
Might be related to #1597 as well?

**Comment by fifoosid** [comment]
Yes it is related and should be fixed with #1622.
